django/test/utils.py

```diff
@@ -347,7 +347,11 @@ def setUp(inner_self):
                 context = self.enable()
                 if self.attr_name:
                     setattr(inner_self, self.attr_name, context)
-                decorated_setUp(inner_self)
+                try:
+                    decorated_setUp(inner_self)
+                except:
+                    self.diable()
```

Comment:
should be `disable`

Reply:
Thanks! I'll be updating the pull request soon.